### ISSUE TITLE
Div scrolls up on focus, using latest version and autoScrollOnFocus

### ISSUE THREAD

**Issue by thiskbj** [malihu/malihu-custom-scrollbar-plugin#127]
Hi there,

I love this plugin, thank you! :) 

I have one thing with it I'm trying to work out. If I scroll down a bit, then click on a textarea, the window scrolls up. I thought this option here would address it but it doesn't:

```
$("div.slide-outside").mCustomScrollbar({
    advanced:{
    autoScrollOnFocus: false
    }
});
```

I've verified that I'm running the latest version of the plugin, I just updated it to be sure. What am I missing here?

Thank you!
Katie

**Comment by rabbit-fdeus** [comment]
Hello!
I'm also having the same issue, but in my case, the scrollbar also gets hidden (visibility is set to none).

I also tried to define the fields I'd like this behaviour to happen, but then the fields are moved to the top left. Is there a way to control the position?

Thank you in advance!
Francisco

**Comment by malihu** [comment]
When `autoScrollOnFocus` is enabled (default) but its scroll-to position is wrong, there's usually some CSS rule missing. For example, are the fields or their parent element positioned absolutely? Maybe a wrapper element with `position:relative;` fixes the issue. 

When `autoScrollOnFocus` is disabled (as @katieben) the scroll-to functionality won't happen but the fields, textarea etc. elements can still be focused with the TAB key. This will result in @rabbit-fdeus case. To disable TAB-focus in HTML you need to add `tabindex="-1"` in your element(s), for instance: `<textarea tabindex="-1">...</textarea>`

**Comment by rabbit-fdeus** [comment]
@malihu Thanks for your response.
Is it possible to do smooth scrolling when `autoScrollOnFocus` is defined?

**Comment by malihu** [comment]
Not really. The scroll-to focused element replicates exactly browser's default scrollbar (instant scroll with no inertia).

**Comment by kargal** [comment]
Hi,
For test purposes I created an empty div with scrollbars. (I am using angular directive ng-scrollbars)
When I hit tab the div get focused which causes the page to scroll.
This doesn't happen when scrollbars are not added.
I tried setting autoScrollOnFocus and temporary disabling the scrollbar but nothing helped.
Any suggestions?
